src/components/ConfigProvider/Theming/styleGenerator.ts

```diff
@@ -242,6 +251,12 @@ export function injectCSS(
 
 export function registerTheme(themeOptions: ThemeOptions): IRegisterTheme {
   const { themeName, light, variables } = getStyle(themeOptions);
+  const body: HTMLElement = document.body;
+  if (themeOptions.dark) {
+    body.setAttribute('dark', '');
+  } else {
+    body.removeAttribute('dark');
+  }
```

Comment:
would it be more sensible to apply this attribute (or a class) at the config provider level? as-is, we can have nested ConfigProviders trying to manage the same attribute

Reply:
Will update

Reply:
The other problem with this approach is other themes, as well as custom themes are currently overridden.